Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebView.cpp

```diff
@@ -2053,6 +2093,7 @@ void beforeAll()
     WebViewTest::add("WebKitWebView", "default-content-security-policy", testWebViewDefaultContentSecurityPolicy);
     WebViewTest::add("WebKitWebView", "web-extension-mode", testWebViewWebExtensionMode);
     WebViewTest::add("WebKitWebView", "disable-web-security", testWebViewDisableWebSecurity);
+    WebViewTest::add("WebKitWebView", "handle-corrupted-local-storage", testWebViewHandleCorruptedLocalStorage);
```

Comment:
So this doesn't really fit amazingly well into any existing tests, but I think I would shoehorn it into TestWebsiteData.cpp instead of here.